docs/data/osdf/install-cache-container.md

```diff
@@ -0,0 +1,347 @@
+
+* __Container runtime:__ The examples in this guide assume using Docker as the container runtime,
+    for which the host must have a running `docker` service,
+   and you must have the ability to start containers (i.e., belong to the `docker` Unix group).
+* __File systems:__ The cache should have a partition of its own for storing data and metadata.
+* __Host certificate:__ Required for authentication.  See note below.
+* __Network ports:__ The cache service requires the following open ports:
+  * Inbound TCP port 8443 for authenticated file access via the HTTP(S) and XRoot protocols.
+  * (Optional) Inbound TCP port 8444 for access to the web interface for monitoring and configuration;
+    if enabled, access to this port should be restricted to the LAN.
+  * Outbound UDP port 9930 for reporting to `xrd-report.osgstorage.org` and `xrd-mon.osgstorage.org` for monitoring
+* __Service requirements:__
+  * A cache serving the OSDF federation as a regional cache should have at least:
+    * 8 cores
+    * 40 Gbps connectivity
+    * 50-200 TB of NVMe disk for the cache partition; you may distribute the disk, e.g., by using an NVMe-backed Ceph pool,
+        if you cannot fit that much disk into a single chassis
+    * 24 GB of RAM
+  * A cache being used to serve data from the OSDF to a single site should have at least:
+    * 8 cores
+    * 40 Gbps connectivity
+    * 2 TB of NVMe disk for the cache partition
+    * 24 GB of RAM
+
+
+!!! note "Host certificates"
+    Caches are accessed by users through browsers, meaning caches need a certificate from a CA acceptable to a standard browser.
+    Examples include [Let's Encrypt](../../security/host-certs/lets-encrypt.md) or the InCommon RSA CA.
+    Caches without a valid certificate for the browser cannot be added to the OSDF.
+    Note that, unlike legacy grid software, the public certificate file will need to contain the "full chain", including any
+    intermediate CAs (if you're unsure about your setup, try accessing your cache from your browser).
+
+    The certificate chain and key should be mounted into the following locations:
+
+    * **Host Certificate Chain**: `/etc/pelican/certificates/tls.crt`
+    * **Host Key**: `/etc/pelican/certificates/tls.key`
+
+
+!!! note "osdf-cache 7.12"
+    This document covers versions 7.12 and later of the `osdf-cache` container image.
+
+!!! note "root"
+    The paths used as examples on this page (e.g., `/etc/pelican`) require root to edit;
+    if you do not have root on the host, modify the directories to a path you do have access to.
+
+
+Configuring the Cache Server
+----------------------------
+
+The preferred method of configuring a Pelican-based OSDF cache is via a YAML file that will be 
+mounted into the container into the `/etc/pelican/config.d` directory.
+This document will use `/etc/pelican/cache-config.yaml` as the name of the configuration file outside of the container.
```

Comment:
I think this gets confusing for host vs container paths: maybe we should pick a host path prefix that's very dissimilar to the container path?

Reply:
Makes sense; what do you suggest?

- `/etc/osdf-cache`
  - `/etc/osdf-cache/host{cert,key}.pem`
  - `/etc/osdf-cache/issuer.jwk`
  - `/etc/osdf-cache/config.yaml` (can rename it from `cache-config.yaml` to avoid redundancy)

- `/etc/osdf-container`
  - `/etc/osdf-container/host{cert,key}.pem`
  - `/etc/osdf-container/issuer.jwk`
  - `/etc/osdf-container/cache-config.yaml`

- `/etc/pelican-container`
  - `/etc/pelican-container/host{cert,key}.pem`
  - `/etc/pelican-container/issuer.jwk`
  - `/etc/pelican-container/cache-config.yaml`

something else?